app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -96,22 +106,32 @@ else
 fi
 <% end -%>
 
+<%
+if @download_utility == 'curl'
+  data_keyword = "--data"
+  registration_command="curl --silent --show-error --request POST --cacert"
+else
+  data_keyword = "--post-data"
```

Comment:
Nit: `data_key` is sufficient in this case

Reply:
Applied